src/Host.cpp

```diff
@@ -456,6 +461,22 @@ void Host::reenableAllTriggers()
     mTimerUnit.reenableAllTriggers();
 }
 
+QPair<QString, QString> Host::getSearchEngine()
+{
+    QPair<QString, QString> ret = QPair<QString, QString>();
+    ret.first = QString("Google");
+    ret.second = mSearchEngineData[ret.first];
+
+    auto iterator = mSearchEngineData.find(mSearchEngineName);
```

Comment:
`constFind` would be a better variant to use as you're not planning on modifying the value iterator points to!